db/migrate/20250318024953_copy_solved_topic_custom_field_to_discourse_solved_solved_topics.rb

```diff
@@ -33,11 +36,9 @@ def up
           AND ua.action_type = 15
         WHERE tc.name = 'accepted_answer_post_id'
           AND tc.id > :last_id
-        ORDER BY tc.topic_id, ua.created_at DESC
-        LIMIT :batch_size
+          AND tc.id <= :last_id + :batch_size
```

Comment:
This doesn't look right 🤔 We keep increasing `last_id` by `batch_size` each loop and then we can looking at rows below a larger `id` with each loop. 

First loop: tc.id <= 10000
Second loop: tc.id <= 20000
etc...

Reply:
```suggestion
          AND tc.id >= :last_id
          AND tc.id <= :last_id + :batch_size
```

I think it should be something like this.